tabs/src/components/sample/SampleQueries.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from "react";
+import { Table, TableCell, OpenOutsideIcon, Button} from '@fluentui/react-northstar';
+import { GRAPH_URL, SAMPLE_QUERIES_URL} from "./TabConstants";
+import * as microsoftTeams from "@microsoft/teams-js";
+/* eslint-disable react/prop-types */
+
+export function FetchSamples(props){
+    const [samples, setSamples] = useState([]);
+    useEffect(() => {
+        async function getSamplesList(FetchSamples) {
+            microsoftTeams.getContext(async function (context) {
+                const devxApi = SAMPLE_QUERIES_URL;
+
+                const headers = {
+                    'Content-Type': 'application/json'
+                };
+    
+                try {
+                        const response = await fetch(devxApi, headers);
+                        if (!response.ok) {
+                            return;
+                    }
+                    const res = await response.json();
+                    var arr = [];
+                    for(let i = 0; i < res.teamsAppSampleQueries.length; i++){
+                        //filter through all the chats and teams
+                        if( (context?.groupId && res.teamsAppSampleQueries[i]["requestUrl"].includes("chat"))||(context?.chatId &&  res.teamsAppSampleQueries[i]["requestUrl"].includes("teams")) ){
+                            continue;
+                        }
+                        // eslint-disable-next-line no-loop-func
```

Comment:
why was this ignored?

Reply:
Oop, saw this too late but it's already fixed! And yea sounds good! I will align to the left with my next commit

Reply:
left align 18b949a8384f973b0f17fdaf8bdfad4c52261944